src/Composer/Autoload/Classmap.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace CoenJacobs\Mozart\Composer\Autoload;
+
+class Classmap implements Autoloader
+{
+    /** @var array */
+    public $paths = [];
```

Comment:
Classmap autoloader should support having files in the array, not just directories. Potentially look into splitting the `$paths` variable up in `$directories` and `$files`.